packages/desktop-client/src/components/reports/reports/CustomReport.jsx

```diff
@@ -230,6 +230,67 @@ export function CustomReport() {
     return null;
   }
 
+  const defaultModeItems = (graph, item) => {
+    const chooseGraph = graph || graphType;
+    const newGraph = disabledList.modeGraphsMap
+      .get(item)
+      .includes(graph || graphType)
+      ? defaultsList.modeGraphsMap.get(item)
+      : chooseGraph;
+    if (disabledList.modeGraphsMap.get(item).includes(graphType)) {
+      setGraphType(newGraph);
+    }
+
+    if (disabledList.graphSplitMap.get(item).get(newGraph).includes(groupBy)) {
+      setGroupBy(defaultsList.graphSplitMap.get(item).get(newGraph));
+    }
+
+    if (
+      disabledList.graphTypeMap.get(item).get(newGraph).includes(balanceType)
+    ) {
+      setBalanceType(defaultsList.graphTypeMap.get(item).get(newGraph));
+    }
+  };
+
+  const defaultItems = item => {
+    const chooseGraph = ReportOptions.groupBy.includes(item) ? graphType : item;
```

Comment:
`ReportOptions.groupBy` is an array of `{description: '...'}` so this is currently throwing an error